src/VirtualClient/VirtualClient.Main/profiles/PERF-GPU-MLPERF-TRAINING.json

```diff
@@ -0,0 +1,95 @@
+{
+    "Description": "MLPerf Training GPU Performance Workload",
+    "Metadata": {
+        "RecommendedMinimumExecutionTime": "10:00:00",
+        "SupportedPlatforms": "linux-x64",
+        "SupportedOperatingSystems": "Ubuntu",
+        "SpecialRequirements": "This is an Nvidia GPU-specialized workload. It depends upon the system having an Nvidia GPU card/chip. Requires an OSDisk of >=256GB size and Data disk of >=8TB size"
+    },
+    "Parameters": {
+        "Username": "",
+        "DiskFilter": "SizeGreaterThan:8000gb",
+        "LinuxCudaVersion": "12.0",
+        "LinuxDriverVersion": "525",
+        "LinuxLocalRunFile": "https://developer.download.nvidia.com/compute/cuda/12.0.0/local_installers/cuda_12.0.0_525.60.13_linux.run"
+    },
+    "Actions": [
+        {
+            "Type": "MLPerfTrainingExecutor",
+            "Parameters": {
+                "Scenario": "training-mlperf-bert-batchsize-45-gpu-8",
+                "Model": "bert",
```

Comment:
Are we onboarding bert only for now? Do we have any research work done on how many models are supported and what all can be onboarded in future so that we can prioritize accordingly. It would be excellent if we can add this information in our documentation.

Reply:
I have added not supported models 
We can prioritize ones which are already present in inference side and gpt.